frontend/app-development/features/overview/components/Navigation/Navigation.test.tsx

```diff
@@ -20,7 +23,9 @@ describe('Navigation', () => {
     });
 
     getFilteredTopBarMenu(RepositoryType.App)
-      .filter((item) => item.key !== TopBarMenu.About)
+      .filter(
+        (item) => item.key !== HeaderMenuItemKey.About && item.key !== HeaderMenuItemKey.Deploy,
```

Comment:
What was the reason to add a filter on `Deploy` here`? 🤔

Reply:
To prevent it from appearing on this page: 
<img width="945" alt="Skjermbilde 2024-09-25 kl  16 51 39" src="https://github.com/user-attachments/assets/6b04243c-7122-4179-8d24-7f3ff8649242">

However, I believe it would be better to move it to utils and give it a better name, so I've done that 😄